test/input.txt

```diff
@@ -96,7 +107,7 @@
   find bet
   # does not match if none have keyword
   find 23912039120
-  # matching should be case-sensitive
+  # matching should be case-insensitive
   find betsy
```

Comment:
is this sufficient to show case insensitivity?

Reply:
I have added an extra test case to check for multiple case-insensitive characters within a name itself, and committed/updated the PR together with a newly updated User Guide and cleaned up code portions you have pointed out.